spec/requests/projects_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ProjectsController, type: :request do
+
+  describe 'GET /projects/:id/use_as_template' do
+    let!(:submitter) { create :user }
+    let!(:season)    { nil }
+    let!(:project)   { create :project, season: season, submitter: submitter }
```

Comment:
If you already use `!` for the project, there's no need for *banging*  the submitter and the season.